libevm/JitVM.cpp

```diff
@@ -21,9 +21,9 @@ inline evm_uint160be toEvmC(Address _addr)
 	return *reinterpret_cast<evm_uint160be*>(&_addr);
 }
 
-inline Address fromEvmC(evm_uint160be _addr)
+inline Address fromEvmC(evm_uint160be const* _addr)
```

Comment:
Could we pass the const reference here (and in `asUint`) and reinterpret_cast the reference?
I think it'd be cleaner to dereference the pointer on the calling side, the thing like `asUint(ptr)` gave me a microsecond of confusion, it's not immediately clear that the function converts the value pointed to and not the pointer